packages/cactus-plugin-keychain-aws-sm/package.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "@hyperledger/cactus-plugin-keychain-aws-sm",
+  "version": "0.6.0",
+  "description": "A keychain implementation storing its entries in AWS Secret Manger.",
+  "main": "dist/lib/main/typescript/index.js",
+  "mainMinified": "dist/cactus-plugin-keychain-aws-sm.node.umd.min.js",
+  "browser": "dist/cactus-plugin-keychain-aws-sm.web.umd.js",
+  "browserMinified": "dist/cactus-plugin-keychain-aws-sm.web.umd.min.js",
+  "module": "dist/lib/main/typescript/index.js",
+  "types": "dist/types/main/typescript/index.d.ts",
+  "files": [
+    "dist/*"
+  ],
+  "scripts": {
+    "generate-sdk": "openapi-generator-cli generate -i ./src/main/json/openapi.json -g typescript-axios -o ./src/main/typescript/generated/openapi/typescript-axios/",
+    "tsc": "tsc --project ./tsconfig.json",
```

Comment:
@jagpreetsinghsasan The `tsc` script can be safely deleted now with the new build system in place.